bundles/org.eclipse.osgi.service.http.servlet.contract/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: JavaServlet OSGi Contract Provider for Equinox
+Bundle-SymbolicName: org.eclipse.osgi.service.http.servlet.contract;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Fragment-Host: 
+ jakarta.servlet-api;bundle-version="[4.0.0,5.0.0)",
+ javax.servlet-api;bundle-version="[3.0.0,5.0.0)",
+ org.eclipse.jetty.servlet-api;bundle-version="[4.0.0,5.0.0)"
+Automatic-Module-Name: org.eclipse.osgi.service.http.servlet.contract
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Provide-Capability: osgi.contract;osgi.contract=JavaServlet
+ ;version:List<Version>="2.6,3.0,3.1,4.0"
+ ;uses:="javax.servlet,javax.servlet.http,javax.servlet.descriptor,javax.servlet.annotation"
+Export-Package: 
+ javax.servlet;version="2.999",
```

Comment:
The real providers have
```suggestion
 javax.servlet;version="2.999";uses:="javax.servlet.annotation,javax.servlet.descriptor",
```
But the two used packages do not exist in javax.servlet 2.2

Reply:
Applied this now.
Additionally used packages should not harm consumers of the servlet version 2 package since they can't use it.